recipe/meta.yaml

```diff
@@ -441,6 +453,7 @@ outputs:
     test:
       commands:
         - test -f $PREFIX/lib/libonedal_sycl.a                     # [linux]
+        # 2023/4/13: 2023.1.0-intel_43558 for osx-64 dooes not have libonedal_sycl.a, see https://anaconda.org/intel/dal-devel/files?version=2023.1.1
```

Comment:
Can you modify this comment to say that this specific version does not provide SYCL support for `osx-64`? We don't want anyone else who's reading this to think that the omission of the library is a mistake of some sort.

Reply:
Modified